### PR TITLE
Update scala examples

### PR DESCRIPTION
**Pull request by Aryex**
### Summary

Some fixes to the scala example folder:
- Added a script for Kerberos example
- Fixed `submit-example.sh` with the correct jar name
- updated readme with procedure for using a modified connector jar

### Additional Reviewers
@jeremyp-bq 
@alexey-temnikov 
@alexr-bq 
@jonathanl-bq

### CODE REVIEW

File: examples/scala/src/main/scala/example/examples/BasicReadWriteExamples.scala [hunk 1]
```diff
@@ -49,7 +49,7 @@ class BasicReadWriteExamples(spark: SparkSession) {
   /**
    * A simple example demonstrating how to write into Vertica, then read back the data in a dataframe.
    * */
-  def writeThenRead(): Unit = {
+  def writeThenReadWithHDFS(): Unit = {
```
Comment (jeremyp-bq) [inline examples/scala/src/main/scala/example/examples/BasicReadWriteExamples.scala:52]: I think it is OK to leave it as writeThenRead as HDFS/Hadoop is the default case.  If you do rename it there are references in the README that also need to be updated.

Comment (Aryex) [reply examples/scala/src/main/scala/example/examples/BasicReadWriteExamples.scala]: updated.

File: examples/scala/src/main/scala/example/Main.scala [hunk 2]
```diff
@@ -22,6 +22,7 @@ object Main {
 
     // Define a Spark master here
     val spark = SparkSession.builder()
+      // .master("local[*]")
```
Comment (jeremyp-bq) [inline examples/scala/src/main/scala/example/Main.scala:25]: Should we uncomment this or remove it?

Comment (Aryex) [reply examples/scala/src/main/scala/example/Main.scala]: removed.

**Review by jeremyp-bq — APPROVED** [review]

**Pull request merged**